controllers/ContactUsController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Request\ContactUsRequest;
+// use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Contracts\View\View;
+use Illuminate\Mail\Message;
+
+class ContactUsController extends Controller
+{
+    public function view(){
+        return view('contacts.contactUs');
+    }
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        // dd($request->validated());
+
+        // $callback = function (string $test): string {
```

Comment:
preferable to remove this code.

Reply:
deleted, in the next part, you can see it https://github.com/liubamiron/bootcamp-project/pull/10/commits/c53453f1ee5fefd6fe7897d81b04a9322f8b41e8.